plugins/generator-1.21.1/datapack-1.21.1/mappings/blocksitems.yaml

```diff
@@ -4070,7 +4070,7 @@ Items.SPECKLED_MELON:
   - "glistering_melon_slice"
 Items.SPAWN_EGG:
   - Items.CREEPER_SPAWN_EGG
-  - "#forge:eggs"
+  - "#c:eggs"

```

Comment:
From documentation & testing, this tag is actually for chicken eggs, not spawn eggs

Reply:
Oops :D

Reply:
Seems such tag no longer exists